tasks/elasticsearch-config.yml

```diff
@@ -24,6 +24,18 @@
   template: src=elasticsearch.j2 dest={{ default_file }} mode=0644 force=yes
   notify: restart elasticsearch
 
+#Copy the systemd specific file if systemd is installed
+- name: Make sure destination dir exists
+  file: path={{ sysd_config_file | dirname }} state=directory recurse=yes mode=0755
```

Comment:
this task should also be apply only with `when: use_system_d and m_lock_enabled` condition

Reply:
Fixed with https://github.com/elastic/ansible-elasticsearch/pull/595/commits/5994840c75320e460056dd44b8af021d389adf0f